src/Juvix/Compiler/Core/Translation/FromSource.hs

```diff
@@ -25,16 +25,16 @@ import Text.Megaparsec qualified as P
 -- | Note: only new symbols and tags that are not in the InfoTable already will be
 -- generated during parsing
 runParser :: Path Abs File -> ModuleId -> InfoTable -> Text -> Either MegaparsecError (InfoTable, Maybe Node)
-runParser fileName mid tab input =
+runParser fileName mid tab input_ =
```

Comment:
Name change

Reply:
This change was moved to https://github.com/anoma/juvix/pull/2582